omero/developers/whatsnew.rst

```diff
@@ -19,20 +19,21 @@ Python BlitzGateway
 - See :doc:`/sysadmins/admins-with-restricted-privileges` to know which privileges are required for which actions.
 
 
-Graphs
-^^^^^^
-
-
 Java Gateway
```

Comment:
This heading can be removed too if there's no content for it

Reply:
I hope for content: https://trello.com/c/szy4IbD2/156-document-gateway-enhancements.

Reply:
👍